troposphere/static/js/components/admin/ResourceRequest/Label.react.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const Label = React.createClass({
+    render() {
+        let style = Object.assign({
+            margin: "7px 0px",
+        }, this.props.style);
+
+        let props = Object.assign({}, this.props, {
+            style
+        });
+
+        return (
+        <label { ...props }></label>
```

Comment:
I don't mean to nitpick, I just wanted to point it out for anybody that might not know about this change in React 15+. 

 ` { ...props }` on html elements should look a little suspicious. On elements like `label` it will probably never come up but we should be aware that higher order components like `react-router` apply props to children that we might take for granted and could confuse someone else down the line if they followed this pattern on a parent element.